cns/restserver/util.go

```diff
@@ -300,8 +331,33 @@ func (service *HTTPRestService) attachOrDetachHelper(req cns.ConfigureContainerN
 	}
 
 	existing, ok := service.getNetworkContainerDetails(cns.SwiftPrefix + req.NetworkContainerid)
+	if service.ChannelMode == cns.Managed && operation == attach {
+		if ok {
+			if existing.WaitingForUpdate {
+				_, returnCode, message := isNCWaitingForUpdate(existing.CreateNetworkContainerRequest.Version, req.NetworkContainerid)
+				if returnCode != Success {
+					return cns.Response{
+						ReturnCode: returnCode,
+						Message:    message}
+				}
+			}
+		} else {
+			var (
+				dncEP     = service.GetOption(acn.OptPrivateEndpoint).(string)
+				infraVnet = service.GetOption(acn.OptInfrastructureNetwork).(string)
+				nodeID    = service.GetOption(acn.OptNodeID).(string)
+			)
+
+			returnCode, msg := service.SyncNodeStatus(dncEP, infraVnet, nodeID, json.RawMessage{})
```

Comment:
can you add comment why are we passing empty podcontext

Reply:
only CNI passes non-empty, there's a func comment about this already